### PR TITLE
Bugfix report time

### PR DESCRIPTION
**Pull request by hans-integral**
Worked with Amber on this.

### CODE REVIEW

File: src/TimeTracker.Data/Models/TimeEntry.cs [hunk 1]
```diff
@@ -19,14 +19,14 @@ public class TimeEntry
         public int? BillingClientId { get; set; }
         public int? ProjectId { get; set; }
         public Guid UserId { get; set; }
+        public TimeEntryTypeEnum TimeEntryType { get; set; }
```
Comment (JamesReate) [inline src/TimeTracker.Data/Models/TimeEntry.cs]: 👍

File: test/TimeTracker.Library.Test/Services/Orchestration/DeleteTests.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            database = new InMemoryDatabaseWithProjectsAndUsers().Database;
+            orchestrator = new SlackMessageOrchestrator(database);
+        }
+
+        [Fact]
+        public async Task HandleCommand_deleteHours_returnsDeletedMessage_andDeletesHours()
+        {
+            var user = database.Users.First();
+            var timeEntryService = new TimeEntryService(user.UserId, database);
+            await timeEntryService.CreateBillableTimeEntry(DateTime.UtcNow.Date, 7, 1, 1);
```
Comment (JamesReate) [inline test/TimeTracker.Library.Test/Services/Orchestration/DeleteTests.cs]: what if person has more than one entry? we should make sure delete removes all entries for day...

File: test/TimeTracker.Library.Test/Services/Orchestration/ReportTests.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+        [Theory]
+        [InlineData(TimeEntryTypeEnum.NonBillable, "Other Non-billable")]
+        [InlineData(TimeEntryTypeEnum.Sick, "Sick")]
+        [InlineData(TimeEntryTypeEnum.Vacation, "Vacation")]
+        [InlineData(TimeEntryTypeEnum.BillableProject, "Billable")]
+        public async Task WhenTimeEntryStoredOnFirstDayOfYear_ThenItGetsDisplayedInReportYtd(TimeEntryTypeEnum entryType, string reportText)
+        {
+            var user = database.Users.First();
+            await StoreTimeEntryOnFirstDayOfMonth(entryType, user, 8, 1);
```
Comment (JamesReate) [inline test/TimeTracker.Library.Test/Services/Orchestration/ReportTests.cs]: 👍

File: test/TimeTracker.Library.Test/Services/Orchestration/DeleteTests.cs (hunk 2 above)
Comment (hans-integral) [reply test/TimeTracker.Library.Test/Services/Orchestration/DeleteTests.cs]: We just refactored this test to be in a separate class. We can add this functionality for sure, but that's not really part of the bugfix.

Comment (JamesReate) [reply test/TimeTracker.Library.Test/Services/Orchestration/DeleteTests.cs]: that's cool, just wanted to be sure to address something when i noticed it

**Pull request merged**